tests/test_enterprise/api_client/test_enterprise_catalog.py

```diff
@@ -0,0 +1,122 @@
+# -*- coding: utf-8 -*-
+"""
+Tests for enterprise.api_client.lms.py
+"""
+from __future__ import absolute_import, unicode_literals, with_statement
+
+import json
+
+import mock
+import requests
+import responses
+
+from enterprise.api_client import enterprise_catalog
+
+TEST_ENTERPRISE_ID = '1840e1dc-59cf-4a78-82c5-c5bbc0b5df0f'
+TEST_ENTERPRISE_CATALOG_UUID = 'cde8287a-1001-457c-b9d5-f9f5bd535427'
+
+def _url(path):
+    """
+    Build a URL for the relevant API named by base_name.
+
+    Args:
+        base_name (str): A name to identify the root URL by
+        path (str): The URL suffix to append to the base path
+    """
+    return requests.compat.urljoin(enterprise_catalog.EnterpriseCatalogApiClient.API_BASE_URL, path)
+
+
```

Comment:
i'm curious about why we dont use the standard test class format here

Reply:
i copied this from another api_client test file, and i'm not sure why it was implemented this way before... 🤷‍♀